### PR TITLE
Restart

### PR DESCRIPTION
**Pull request by wcpines**

### CODE REVIEW

File: data/topics.yml [hunk 1]
```diff
@@ -106,6 +106,8 @@
       url: "paas/how-to-update-ssl-certificate"
     - title: "How can I use Aptible’s OSS Docker images?"
       url: "paas/aptible-docker-images"
+    - title: "How can I automatically restart my app after a crash"
```
Comment (fancyremarker) [inline data/topics.yml]: Punctuation: ?

File: source/topics/paas/how-to-auto-restart-your-app.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+There are a few different stopgaps you might consider implementing to automatically restart an app after it crashes.
+
+1. One lightweight method that is framework-agnostic is to just wrap your service definitions in a bash `while` loop. Your Procfile might look like:
+
+  ```bash
+  web: while true; do bundle exec rails s; sleep 1; done
```
Comment (fancyremarker) [inline source/topics/paas/how-to-auto-restart-your-app.md]: Let's make this consistent with below: `bundle exec rails server`

File: source/topics/paas/how-to-auto-restart-your-app.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  ```
+
+  or
+
+  ```bash
+  web: while true; do node server.js; sleep 1; done
```
Comment (fancyremarker) [inline source/topics/paas/how-to-auto-restart-your-app.md]: Same as previous comment: `node app.js`

**Comment by wcpines** [comment]
Done, rebased.

**Comment by fancyremarker** [comment]
Thanks!

**Pull request merged**